apps/judicial-system/web/src/routes/Court/Indictments/Defender/SelectDefender.tsx

```diff
@@ -10,7 +10,8 @@ import {
 import { FormContext } from '@island.is/judicial-system-web/src/components/FormProvider/FormProvider'
 import { core } from '@island.is/judicial-system-web/messages'
 import { capitalize } from '@island.is/judicial-system/formatters'
-import { Defendant, UpdateDefendant } from '@island.is/judicial-system/types'
+import { TempUpdateDefendant as UpdateDefendant } from '@island.is/judicial-system-web/src/types'
```

Comment:
Will you eventually be using the generated UpdateDefendantInput here?

Reply:
No, probably not! I changed this to use generated `UpdateDefendantInput` and `CreateDefendantInput` instead. Could you have another look?